system_tests/batch_poster_test.go

```diff
@@ -144,22 +144,22 @@ func TestBatchPosterLargeTx(t *testing.T) {
 
 	conf := arbnode.ConfigDefaultL1Test()
 	conf.Sequencer.MaxTxDataSize = 110000
-	l2info, nodeA, l2clientA, l1info, _, _, l1stack := createTestNodeOnL1WithConfig(t, ctx, true, conf, nil, nil)
-	defer requireClose(t, l1stack)
-	defer nodeA.StopAndWait()
+	testNodeA := NewNodeBuilder(ctx).SetNodeConfig(conf).SetIsSequencer(true).CreateTestNodeOnL1AndL2(t)
+	defer requireClose(t, testNodeA.L1Stack)
+	defer testNodeA.L2Node.StopAndWait()
```

Comment:
possibly in a later PR: the builer should have one simple cleanup() function that does all cleanup for nodes / stacks / etc built from inside it automatically.

Reply:
e.g. all nodes generally share l1info / l2info/ l1 node..

Reply:
there is added complexity because different nodes could have different configs, but I think it can / should be solved.
It's fine to have all normal fields/etc have a "default" L2 node and some other fields/functions only used by additional nodes.